networking/multiple_networks/primary_networks/about-user-defined-networks.adoc

```diff
@@ -18,6 +18,8 @@ image::527-OpenShift-UDN-isolation-012025.png[The namespace isolation concept in
 [NOTE]
 ====
 Support for the Localnet topology on both primary and secondary networks will be added in a future version of {product-title}.
+
```

Comment:
Do we mention about event generated to that affect to inform the user?

Reply:
UDNKubeletProbesNotSupported

Reply:
This is outside the scope of this PR.